### DISCUSSION TITLE
Option to copy line left/right

### DISCUSSION THREAD

**Discussion by crakerjac** [WinMerge/winmerge#527] [Ideas]
Can there be an option to copy a specific/selected line left/right vs only being able to copy a whole block?

**Comment by sdottaka** [comment]
WinMerge 2.16.12 now includes a Copy Selected Lines(s) to Right/Left menu item.

![image](https://user-images.githubusercontent.com/98126/133929331-dddaa0ca-e298-4a1f-98ea-be6b25d364dc.png)
